### PR TITLE
Fix missing enum value for `Window.initial_position`

### PR DESCRIPTION
**Pull request by jeanahelver**

### CODE REVIEW

**Comment by clayjohn** [comment]
The substantive change looks great! 

The CI is currently failing on some style checks. It looks like you have left tabs in the newlines which will need to be deleted. You will also need to squash all the commits together so that the whole PR only contains 1 big commit with all your changes. We like to merge one commit at a time to keep the git history clean and navigable. You can do that at the same time as you delete the tabs.

If you don't know how to squash all the commits together, we have a helpful tutorial in the official documentation https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase I highly recommend giving this a quick read and then fixing up your PR and squashing everything together all at once.

File: scene/main/window.cpp [hunk 1]
```diff
@@ -2660,14 +2660,15 @@ void Window::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("popup_exclusive_centered_ratio", "from_node", "ratio"), &Window::popup_exclusive_centered_ratio, DEFVAL(0.8));
 	ClassDB::bind_method(D_METHOD("popup_exclusive_centered_clamped", "from_node", "minsize", "fallback_ratio"), &Window::popup_exclusive_centered_clamped, DEFVAL(Size2i()), DEFVAL(0.75));
 
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "initial_position", PROPERTY_HINT_ENUM, "Absolute,Center of Primary Screen,Center of Other Screen,Center of Screen With Mouse Pointer,Center of Screen With Keyboard Focus"), "set_initial_position", "get_initial_position");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "initial_position", PROPERTY_HINT_ENUM, "Absolute,Center of Primary Screen, Center of Main Screen,Center of Other Screen,Center of Screen With Mouse Pointer,Center of Screen With Keyboard Focus"), "set_initial_position", "get_initial_position");
```
Comment (akien-mga) [inline scene/main/window.cpp:2663]:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::INT, "initial_position", PROPERTY_HINT_ENUM, "Absolute,Center of Primary Screen,Center of Main Screen,Center of Other Screen,Center of Screen With Mouse Pointer,Center of Screen With Keyboard Focus"), "set_initial_position", "get_initial_position");
```

**Comment by akien-mga** [comment]
I went ahead and did the requested changes myself, plus a few more, as I noticed a comment was out of date and the actual important comments were missing.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Pull request merged**